keyboards/tdcleft104/info.json

```diff
@@ -0,0 +1,119 @@
+            {"label":"E", "x":7.5, "y":2.5}, 
+            {"label":"R", "x":8.5, "y":2.5}, 
+            {"label":"T", "x":9.5, "y":2.5}, 
+            {"label":"Y", "x":10.5, "y":2.5}, 
+            {"label":"U", "x":11.5, "y":2.5}, 
+            {"label":"I", "x":12.5, "y":2.5}, 
+            {"label":"O", "x":13.5, "y":2.5}, 
+            {"label":"P", "x":14.5, "y":2.5}, 
+            {"label":"{", "x":15.5, "y":2.5}, 
+            {"label":"}", "x":16.5, "y":2.5}, 
+            {"label":"|", "x":17.5, "y":2.5, "w":1.25},
+            {"label":"F16", "x":18.75, "y":2.5}, 
+            {"label":"4", "x":0, "y":3.5}, 
+            {"label":"5", "x":1, "y":3.5}, 
+            {"label":"6", "x":2, "y":3.5}, 
+            {"label":"Caps Lock", "x":4.25, "y":3.5, "w":1.25}, 
+            {"label":"A", "x":5.75, "y":3.5}, 
+            {"label":"S", "x":6.75, "y":3.5}, 
+            {"label":"D", "x":7.75, "y":3.5}, 
+            {"label":"F", "x":8.75, "y":3.5}, 
+            {"label":"G", "x":9.75, "y":3.5}, 
+            {"label":"H", "x":10.75, "y":3.5}, 
+            {"label":"J", "x":11.75, "y":3.5}, 
+            {"label":"K", "x":12.75, "y":3.5}, 
+            {"label":"L", "x":13.75, "y":3.5}, 
+            {"label":":", "x":14.75, "y":3.5}, 
+            {"label":"|", "x":15.75, "y":3.5}, 
+            {"label":"Enter", "x":16.75, "y":3.5, "w":2}, 
+            {"label":"F14", "x":18.75, "y":3.5}, 
+            {"label":"1", "x":0, "y":4.5}, 
+            {"label":"2", "x":1, "y":4.5}, 
+            {"label":"3", "x":2, "y":4.5}, 
+            {"label":"Enter", "x":3, "y":4.5, "h":2}, 
+            {"label":"Shift", "x":4.25, "y":4.5, "w":1.75}, 
+            {"label":"Z", "x":6, "y":4.5}, 
+            {"label":"X", "x":7, "y":4.5}, 
+            {"label":"C", "x":8, "y":4.5}, 
+            {"label":"V", "x":9, "y":4.5}, 
+            {"label":"B", "x":10, "y":4.5}, 
+            {"label":"N", "x":11, "y":4.5}, 
+            {"label":"M", "x":12, "y":4.5}, 
+            {"label":"<", "x":13, "y":4.5}, 
+            {"label":">", "x":14, "y":4.5}, 
+            {"label":"?", "x":15, "y":4.5}, 
+            {"label":"Shift", "x":16, "y":4.5, "w":1.5}, 
+            {"label":"u2191", "x":17.5, "y":4.5, "w":1.25}, 
+            {"label":"F15", "x":18.75, "y":4.5}, 
+            {"label":"0", "x":0, "y":5.5, "w":2}, 
+            {"label":".", "x":2, "y":5.5}, 
+            {"label":"Ctrl", "x":4.25, "y":5.5}, 
+            {"label":"Fn", "x":5.25, "y":5.5}, 
+            {"label":"<i class='kb kb-logo-windows-8'></i>", "x":6.25, "y":5.5}, 
+            {"label":"Alt", "x":7.25, "y":5.5}, 
+            {"x":8.25, "y":5.5, "w":6.25}, 
+            {"label":"Alt", "x":14.5, "y":5.5}, 
+            {"label":"Ctrl", "x":15.5, "y":5.5}, 
+            {"label":"u2190", "x":16.5, "y":5.5}, 
+            {"label":"u2193", "x":17.5, "y":5.5, "w":1.25}, 
+            {"label":"u2192", "x":18.75, "y":5.5}]
+        }
+    }
+{

```

Comment:
```suggestion
```

Reply:
So you want a - in there?

Reply:
No, that is part of the diff. Please just apply the suggestions directly.